clubs/README.md

```diff
@@ -50,66 +49,64 @@ _(each of the sections below corresponds with a bullet point)_
 
 ### Leadership Team
 
-You'll want 2-3 people on the primary leadership team running the club. When
-choosing your leadership team, we recommend working with close friends who you
-think have strong leadership skills. It's also a good idea to have a mix of
-genders on the leadership team. Correlation doesn't imply causation, but the
-club member demographics mirror the demographics of the leadership team in many
-clubs.
+You'll want 2-3 people on the leadership team running the club. When choosing
+your team, we recommend working with close friends who you think have strong
+leadership skills. It's a good idea to have a mix of genders on the team.
+Correlation doesn't imply causation, but the club member demographics mirror the
```

Comment:
Eh – I actually agree with you that it can be better. Let me know if you
like the new version.

On Sun, Jan 17, 2016 at 2:08 AM, Max Wofford <notifications@github.com>
wrote:

> In clubs/README.md
> <https://github.com/hackclub/hackclub/pull/658#discussion_r49944097>:
>
> > @@ -50,66 +49,64 @@ _(each of the sections below corresponds with a bullet point)_
> >
> >  ### Leadership Team
> >
> > -You'll want 2-3 people on the primary leadership team running the club. When
> > -choosing your leadership team, we recommend working with close friends who you
> > -think have strong leadership skills. It's also a good idea to have a mix of
> > -genders on the leadership team. Correlation doesn't imply causation, but the
> > -club member demographics mirror the demographics of the leadership team in many
> > -clubs.
> > +You'll want 2-3 people on the leadership team running the club. When choosing
> > +your team, we recommend working with close friends who you think have strong
> > +leadership skills. It's a good idea to have a mix of genders on the team.
> > +Correlation doesn't imply causation, but the club member demographics mirror the
>
> I take that back. Just reread the paragraph. It's fine.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/hackclub/hackclub/pull/658/files#r49944097>.
>



-- 
Zach Latta
http://zachlatta.com
https://github.com/zachlatta
https://twitter.com/zachlatta